control-station/src/services/PodSocketClient.ts

```diff
@@ -114,6 +120,25 @@ class PodSocketClient {
 	private onData(data: string): void {
 		console.log("server says", data);
 	}
+
+	private addMessage(response: string, newState: State): void {
+		const currentTime = new Date();
+		const hours = currentTime.getHours().toString().padStart(2, "0");
+		const minutes = currentTime.getMinutes().toString().padStart(2, "0");
+		const seconds = currentTime.getSeconds().toString().padStart(2, "0");
+		const timestamp = `${hours}:${minutes}:${seconds}`;
```

Comment:
Suggestion: having the provider dictate the formatting of this value limits the freedom of the consumer. We could store the original `Date` object in the message object and form the string representation when displayed.